### PR TITLE
Update config.yml

### PR DESCRIPTION
**Pull request by ADITYADAS1999**
## Description:

[FEATURE] Add Sema's Discord on the contact_links field #2

### CODE REVIEW

**Review by kaiwalyakoparkar — APPROVED** [review]
lgtm, thank you so much for contributing

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -2,4 +2,4 @@ blank_issues_enabled: false
 contact_links:
   - name: Questions
     url: https://discord.gg/Byjr6rdBUZ
-    about: Any questions or comments you have, let us know in our Discord server.
+    about: Any questions or comments you have, let us know on our Discord server.
```
Comment (naomi-lgbt) [inline .github/ISSUE_TEMPLATE/config.yml:5]: I don't believe this change is correct.

You place a file *on* a computer server, but you interact with people *in* a discord server.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (ADITYADAS1999) [reply .github/ISSUE_TEMPLATE/config.yml:5]: apologize for that, I mention the error in discord.

Comment (kaiwalyakoparkar) [reply .github/ISSUE_TEMPLATE/config.yml:5]: Good point @naomi-lgbt 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient

**Comment by naomi-lgbt** [comment]
Hey there~!

Thanks for your interest in contributing. We're going to close this pull request, because as we discussed on Discord this change is not necessary. We would gladly welcome future contributions from you!

**Pull request closed without merging**